config/locales/user_mailer/es.yml

```diff
@@ -256,11 +255,12 @@ es:
         %{app_name}</a></strong> y asegúrese de reconocer toda la información en
         la página de su cuenta, incluidos los métodos que utiliza para la
         autenticación de dos factores, como el número de teléfono, la aplicación
-        de autenticación, o la clave de seguridad.</li><li><strong>En la página
-        de su cuenta %{app_name}, solicite una nueva clave personal.</strong>
-        Recuerde, nunca lo comparta a menos que lo esté utilizando para iniciar
-        sesión en un sitio web de confianza que utiliza
-        %{app_name}.</li></ol></p><br>Gracias,<br>El equipo de %{app_name}
+        de autenticación, o la clave de seguridad.</li><li><strong>En <a
+        href="%{account_url}">la página de su cuenta %{app_name}</a>, solicite
```

Comment:
English version has a third link "your Login.gov account page", which was not reflected in the other locales:

https://github.com/18F/identity-idp/blob/0f21a2a06f467873b6391dd20621d50c90b4e633/config/locales/user_mailer/en.yml#L242-L243

Reply:
this is a good candidate for one translation string that should be broken into 2-3 and we should move the markup into the template IMO

Reply:
Yeah, the email templates in general could use a lot of love 😬 Not going to try to tackle that here.